### PR TITLE
Update Comet up to v0.7.0

### PR DESCRIPTION
**Pull request by gotzmann**
1) Update Comet up to v0.7.0
2) Change Date header once per second

### CODE REVIEW

File: frameworks/PHP/comet/benchmark_config.json [hunk 1]
```diff
@@ -25,6 +25,8 @@
       "versus": "php"
     },
     "mysql": {
+      "json_url": "/json",
+      "plaintext_url": "/plaintext",
```
Comment (joanhey) [inline frameworks/PHP/comet/benchmark_config.json:29]: Please, delete this 2 tests.
Are the same that with postgres.

File: frameworks/PHP/comet/src/ORM.php [hunk 2]
```diff
@@ -14,8 +14,8 @@ class ORM
     public static function init()
     {
         self::$pdo = new \PDO(
-            'pgsql:host=tfb-database;dbname=hello_world',
-//	    	'pgsql:host=192.168.99.1;dbname=hello_world',
+//            'pgsql:host=tfb-database;dbname=hello_world',
+	    	'pgsql:host=192.168.99.1;dbname=hello_world',
```
Comment (joanhey) [inline frameworks/PHP/comet/src/ORM.php:18]: Change to host=tfb-database

File: frameworks/PHP/comet/benchmark_config.json (hunk 1 above)
Comment (gotzmann) [reply frameworks/PHP/comet/benchmark_config.json]: OK, done

File: frameworks/PHP/comet/src/ORM.php (hunk 2 above)
Comment (gotzmann) [reply frameworks/PHP/comet/src/ORM.php]: My fault, done

**Pull request merged**